### ISSUE TITLE
[Tests] java-network test hw-mode failure

### ISSUE THREAD

**Issue by SeanTAllen** [lsds/sgx-lkl#556]
While testing turning back on some disabled tests, I encountered this error with the languages/java-network test.

I've done no additional triaging yet as it was out of scope for what I was doing.

```
2020-06-26T13:22:14.4768640Z INFO: Junit data recorded to 'report/TEST-languages-java-network-(debug)-(run-hw)-(8-ethreads)-junit.xml'
2020-06-26T13:22:14.4769400Z Test 'languages-java-network-(debug)-(run-hw)-(8-ethreads)' status - failed (Exit code: '2'; FAILURES in output: '1').
2020-06-26T13:22:14.4769976Z Test 'languages-java-network-(debug)-(run-hw)-(8-ethreads)' status - Completed.
2020-06-26T13:22:14.4770285Z Current Status: passed = 24, failed = 1, disabled = 0, remaining = 11.
```

Failure logs are:

- https://dev.azure.com/sgx-lkl/sgx-lkl/_build/results?buildId=989&view=ms.vss-test-web.build-test-results-tab&runId=16266&paneView=debug&resultId=100022
- https://dev.azure.com/sgx-lkl/sgx-lkl/_build/results?buildId=989&view=ms.vss-test-web.build-test-results-tab&runId=16274&paneView=debug&resultId=100022

**Comment by SeanTAllen** [comment]
TIL that build logs go away at some point and in the future I should download them and attach directly to the issue.

**Comment by SeanTAllen** [comment]
This still happens for me.

```
[    0.097667] EXT4-fs (vda): warning: mounting unchecked fs, running e2fsck is recommended
[    0.097667] EXT4-fs (vda): mounted filesystem without journal. Opts:
[[  SGX-LKL ]] lkl_mount_disks(): Set working directory: /
[[  SGX-LKL ]] libc_start_main_stage2(): Calling app main: /usr/bin/java
OpenJDK 64-Bit Server VM warning: Can't detect primordial thread stack location - find_vma failed
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f603d047578, pid=36, tid=0x00007f6000b0b350
#
# JRE version: OpenJDK Runtime Environment (8.0_212-b04) (build 1.8.0_212-b04)
# Java VM: OpenJDK 64-Bit Server VM (25.212-b04 mixed mode linux-amd64 compressed oops)
# Derivative: IcedTea 3.12.0
# Distribution: Custom build (Sat May  4 17:33:35 UTC 2019)
# Problematic frame:
# J 700 C2 java.util.Arrays.equals([B[B)Z (54 bytes) @ 0x00007f603d047578 [0x00007f603d047560+0x18]
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
bad count while changing owner
[[  SGX-LKL ]] FAIL: Kernel panic! Aborting...
[[  SGX-LKL ]] =============================================================
[[  SGX-LKL ]] Stack traces for all lthreads:
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 1: tid=1 (7f6000b05480) [java]
[[  SGX-LKL ]]     #0: 7f60005bfc69 in lthread_join(...)
[[  SGX-LKL ]]     #1: 7f600059a25e in __pthread_join(...)
[[  SGX-LKL ]]     #2: 7f603dcf3eee in <unknown>(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 2: tid=2 (7f6000b05630) [vio-0]
[[  SGX-LKL ]]     #0: 7f60005acac9 in vio_wait_for_host_event(...)
[[  SGX-LKL ]]     #1: 7f60005acc99 in vio_enclave_process_host_event(...)
[[  SGX-LKL ]]     #2: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 3: tid=3 (7f6000b08180) [vio-1]
[[  SGX-LKL ]]     #0: 7f60005acac9 in vio_wait_for_host_event(...)
[[  SGX-LKL ]]     #1: 7f60005acc99 in vio_enclave_process_host_event(...)
[[  SGX-LKL ]]     #2: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 4: tid=4 (7f6000b08330) [vio-2]
[[  SGX-LKL ]]     #0: 7f60005acac9 in vio_wait_for_host_event(...)
[[  SGX-LKL ]]     #1: 7f60005acc99 in vio_enclave_process_host_event(...)
[[  SGX-LKL ]]     #2: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 5: tid=5 (7f6000b084e0) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414fbd in schedule_idle(...)
[[  SGX-LKL ]]     #8: 7f60000b0dea in do_idle(...)
[[  SGX-LKL ]]     #9: 7f60000b1045 in cpu_startup_entry(...)
[[  SGX-LKL ]]     #10: 7f6000412d2e in rest_init(...)
[[  SGX-LKL ]]     #11: 7f6000628756 in arch_call_rest_init(...)
[[  SGX-LKL ]]     #12: 7f6000628af9 in start_kernel(...)
[[  SGX-LKL ]]     #13: 7f60006293e4 in lkl_run_kernel(...)
[[  SGX-LKL ]]     #14: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 6: tid=6 (7f6000b086e0) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005d6687 in timer_callback(...)
[[  SGX-LKL ]]     #4: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 7: tid=7 (7f6000b08890) [kernel]
[[  SGX-LKL ]]     #0: 7f60005bfa55 in lthread_join(...)
[[  SGX-LKL ]]     #1: 7f60005d63fb in thread_exit(...)
[[  SGX-LKL ]]     #2: 7f600008a70e in lkl_run_init(...)
[[  SGX-LKL ]]     #3: 7f600010ffcf in search_binary_handler(...)
[[  SGX-LKL ]]     #4: 7f60001105d5 in __do_execve_file(...)
[[  SGX-LKL ]]     #5: 7f600011070d in do_execve(...)
[[  SGX-LKL ]]     #6: 7f600008a0b2 in run_init_process(...)
[[  SGX-LKL ]]     #7: 7f6000412d6c in kernel_init(...)
[[  SGX-LKL ]]     #8: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #9: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 8: tid=8 (7f6000b08a40) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000a95b5 in kthreadd(...)
[[  SGX-LKL ]]     #9: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #10: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 9: tid=9 (7f6000b08bf0) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000a3d73 in worker_thread(...)
[[  SGX-LKL ]]     #9: 7f60000a8a44 in kthread(...)
[[  SGX-LKL ]]     #10: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #11: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 10: tid=10 (7f6000b08da0) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000a3d73 in worker_thread(...)
[[  SGX-LKL ]]     #9: 7f60000a8a44 in kthread(...)
[[  SGX-LKL ]]     #10: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #11: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 11: tid=11 (7f6000b08f50) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000a3d73 in worker_thread(...)
[[  SGX-LKL ]]     #9: 7f60000a8a44 in kthread(...)
[[  SGX-LKL ]]     #10: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #11: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 12: tid=12 (7f6000b09120) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000a4926 in rescuer_thread(...)
[[  SGX-LKL ]]     #9: 7f60000a8a44 in kthread(...)
[[  SGX-LKL ]]     #10: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #11: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 13: tid=13 (7f6000b09310) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000ac705 in smpboot_thread_fn(...)
[[  SGX-LKL ]]     #9: 7f60000a8a44 in kthread(...)
[[  SGX-LKL ]]     #10: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #11: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 14: tid=14 (7f6000b09500) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f6000255196 in devtmpfsd(...)
[[  SGX-LKL ]]     #9: 7f60000a8a44 in kthread(...)
[[  SGX-LKL ]]     #10: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #11: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 15: tid=15 (7f6000b096f0) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000a4926 in rescuer_thread(...)
[[  SGX-LKL ]]     #9: 7f60000a8a44 in kthread(...)
[[  SGX-LKL ]]     #10: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #11: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 16: tid=34 (7f6000b098e0) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000a4926 in rescuer_thread(...)
[[  SGX-LKL ]]     #9: 7f60000a8a44 in kthread(...)
[[  SGX-LKL ]]     #10: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #11: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 17: tid=35 (7f6000b09ad0) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000a4926 in rescuer_thread(...)
[[  SGX-LKL ]]     #9: 7f60000a8a44 in kthread(...)
[[  SGX-LKL ]]     #10: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #11: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 18: tid=36 (7f6000b09cc0) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f6000417966 in schedule_timeout(...)
[[  SGX-LKL ]]     #9: 7f60000ee524 in kswapd(...)
[[  SGX-LKL ]]     #10: 7f60000a8a44 in kthread(...)
[[  SGX-LKL ]]     #11: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #12: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 19: tid=37 (7f6000b09eb0) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000a3d73 in worker_thread(...)
[[  SGX-LKL ]]     #9: 7f60000a8a44 in kthread(...)
[[  SGX-LKL ]]     #10: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #11: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 20: tid=38 (7f6000b0a0a0) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f600023f285 in khvcd(...)
[[  SGX-LKL ]]     #9: 7f60000a8a44 in kthread(...)
[[  SGX-LKL ]]     #10: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #11: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 21: tid=39 (7f6000b0a290) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000a4926 in rescuer_thread(...)
[[  SGX-LKL ]]     #9: 7f60000a8a44 in kthread(...)
[[  SGX-LKL ]]     #10: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #11: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 22: tid=40 (7f6000b0a480) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000a4926 in rescuer_thread(...)
[[  SGX-LKL ]]     #9: 7f60000a8a44 in kthread(...)
[[  SGX-LKL ]]     #10: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #11: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 23: tid=41 (7f6000b0a6c0) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008b62f in idle_host_task_loop(...)
[[  SGX-LKL ]]     #6: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #7: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 24: tid=42 (7f6000b0a8b0) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f60005d9ca3 in lkl_termination_thread(...)
[[  SGX-LKL ]]     #6: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 25: tid=43 (7f6000b0ad50) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000a3d73 in worker_thread(...)
[[  SGX-LKL ]]     #9: 7f60000a8a44 in kthread(...)
[[  SGX-LKL ]]     #10: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #11: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 26: tid=44 (7f6000b0ab60) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000a4926 in rescuer_thread(...)
[[  SGX-LKL ]]     #9: 7f60000a8a44 in kthread(...)
[[  SGX-LKL ]]     #10: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #11: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 27: tid=45 (7f6000b0af90) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000a3d73 in worker_thread(...)
[[  SGX-LKL ]]     #9: 7f60000a8a44 in kthread(...)
[[  SGX-LKL ]]     #10: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #11: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 28: tid=46 (7f6000b0b180) [kernel]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000a4926 in rescuer_thread(...)
[[  SGX-LKL ]]     #9: 7f60000a8a44 in kthread(...)
[[  SGX-LKL ]]     #10: 7f600008a928 in thread_bootstrap(...)
[[  SGX-LKL ]]     #11: 7f60005be4cd in _exec(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] *29: tid=47 (7f6000b0b350) [java]
[[  SGX-LKL ]]     #0: 7f60005aff76 in sgxlkl_fail(...)
[[  SGX-LKL ]]     #1: 7f60005d5dcf in panic(...)
[[  SGX-LKL ]]     #2: 7f6000085283 in lkl_bug(...)
[[  SGX-LKL ]]     #3: 7f600008aa5c in __switch_to(...)
[[  SGX-LKL ]]     #4: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #5: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #6: 7f6000415275 in io_schedule(...)
[[  SGX-LKL ]]     #7: 7f6000415660 in bit_wait_io(...)
[[  SGX-LKL ]]     #8: 7f60004152f9 in __wait_on_bit(...)
[[  SGX-LKL ]]     #9: 7f60004153de in out_of_line_wait_on_bit(...)
[[  SGX-LKL ]]     #10: 7f600013b46a in __wait_on_buffer(...)
[[  SGX-LKL ]]     #11: 7f60001812e9 in ext4_read_inode_bitmap(...)
[[  SGX-LKL ]]     #12: 7f6000181c07 in __ext4_new_inode(...)
[[  SGX-LKL ]]     #13: 7f60001a8bf2 in ext4_create(...)
[[  SGX-LKL ]]     #14: 7f600011777d in path_openat(...)
[[  SGX-LKL ]]     #15: 7f6000117d83 in do_filp_open(...)
[[  SGX-LKL ]]     #16: 7f60001074ef in do_sys_open(...)
[[  SGX-LKL ]]     #17: 7f600010756f in __se_sys_openat(...)
[[  SGX-LKL ]]     #18: 7f600008b86f in lkl_syscall(...)
[[  SGX-LKL ]]     #19: 7f6000533c4a in __filter_syscall4(...)
[[  SGX-LKL ]]     #20: 7f6000533d7a in open(...)
[[  SGX-LKL ]]     #21: 7f603da0c0d0 in <unknown>(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 30: tid=48 (7f6000b0b5d0) [java]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000d6471 in futex_wait_queue_me(...)
[[  SGX-LKL ]]     #9: 7f60000d7e19 in futex_wait(...)
[[  SGX-LKL ]]     #10: 7f60000d918b in do_futex(...)
[[  SGX-LKL ]]     #11: 7f60000d9955 in sys_futex(...)
[[  SGX-LKL ]]     #12: 7f600008b86f in lkl_syscall(...)
[[  SGX-LKL ]]     #13: 7f6000597af9 in __filter_syscall6(...)
[[  SGX-LKL ]]     #14: 7f6000597c0f in __timedwait_cp(...)
[[  SGX-LKL ]]     #15: 7f6000599b74 in __pthread_cond_timedwait(...)
[[  SGX-LKL ]]     #16: 7f603d94ed8f in <unknown>(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 31: tid=49 (7f6000b0b820) [java]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000d6471 in futex_wait_queue_me(...)
[[  SGX-LKL ]]     #9: 7f60000d7e19 in futex_wait(...)
[[  SGX-LKL ]]     #10: 7f60000d918b in do_futex(...)
[[  SGX-LKL ]]     #11: 7f60000d9955 in sys_futex(...)
[[  SGX-LKL ]]     #12: 7f600008b86f in lkl_syscall(...)
[[  SGX-LKL ]]     #13: 7f6000597af9 in __filter_syscall6(...)
[[  SGX-LKL ]]     #14: 7f6000597c0f in __timedwait_cp(...)
[[  SGX-LKL ]]     #15: 7f6000599b74 in __pthread_cond_timedwait(...)
[[  SGX-LKL ]]     #16: 7f6000599f16 in pthread_cond_wait(...)
[[  SGX-LKL ]]     #17: 7f603d94eba1 in <unknown>(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 32: tid=50 (7f6000b0ba70) [java]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000d6471 in futex_wait_queue_me(...)
[[  SGX-LKL ]]     #9: 7f60000d7e19 in futex_wait(...)
[[  SGX-LKL ]]     #10: 7f60000d918b in do_futex(...)
[[  SGX-LKL ]]     #11: 7f60000d9955 in sys_futex(...)
[[  SGX-LKL ]]     #12: 7f600008b86f in lkl_syscall(...)
[[  SGX-LKL ]]     #13: 7f6000597af9 in __filter_syscall6(...)
[[  SGX-LKL ]]     #14: 7f6000597c0f in __timedwait_cp(...)
[[  SGX-LKL ]]     #15: 7f6000599b74 in __pthread_cond_timedwait(...)
[[  SGX-LKL ]]     #16: 7f6000599f16 in pthread_cond_wait(...)
[[  SGX-LKL ]]     #17: 7f603d94eba1 in <unknown>(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 33: tid=51 (7f6000b0bcc0) [java]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000d6471 in futex_wait_queue_me(...)
[[  SGX-LKL ]]     #9: 7f60000d7e19 in futex_wait(...)
[[  SGX-LKL ]]     #10: 7f60000d918b in do_futex(...)
[[  SGX-LKL ]]     #11: 7f60000d9955 in sys_futex(...)
[[  SGX-LKL ]]     #12: 7f600008b86f in lkl_syscall(...)
[[  SGX-LKL ]]     #13: 7f6000597af9 in __filter_syscall6(...)
[[  SGX-LKL ]]     #14: 7f6000597c0f in __timedwait_cp(...)
[[  SGX-LKL ]]     #15: 7f600059c7a9 in sem_timedwait(...)
[[  SGX-LKL ]]     #16: 7f600059c8c6 in sem_wait(...)
[[  SGX-LKL ]]     #17: 7f603d949f5d in <unknown>(...)
[[  SGX-LKL ]]     #18: 7f602c194000 in <unknown>(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 34: tid=52 (7f6000b0bf10) [java]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000d6471 in futex_wait_queue_me(...)
[[  SGX-LKL ]]     #9: 7f60000d7e19 in futex_wait(...)
[[  SGX-LKL ]]     #10: 7f60000d918b in do_futex(...)
[[  SGX-LKL ]]     #11: 7f60000d9955 in sys_futex(...)
[[  SGX-LKL ]]     #12: 7f600008b86f in lkl_syscall(...)
[[  SGX-LKL ]]     #13: 7f6000597af9 in __filter_syscall6(...)
[[  SGX-LKL ]]     #14: 7f6000597c0f in __timedwait_cp(...)
[[  SGX-LKL ]]     #15: 7f6000599b74 in __pthread_cond_timedwait(...)
[[  SGX-LKL ]]     #16: 7f603d94ed8f in <unknown>(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 35: tid=53 (7f6000b0c160) [java]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000d6471 in futex_wait_queue_me(...)
[[  SGX-LKL ]]     #9: 7f60000d7e19 in futex_wait(...)
[[  SGX-LKL ]]     #10: 7f60000d918b in do_futex(...)
[[  SGX-LKL ]]     #11: 7f60000d9955 in sys_futex(...)
[[  SGX-LKL ]]     #12: 7f600008b86f in lkl_syscall(...)
[[  SGX-LKL ]]     #13: 7f6000597af9 in __filter_syscall6(...)
[[  SGX-LKL ]]     #14: 7f6000597c0f in __timedwait_cp(...)
[[  SGX-LKL ]]     #15: 7f6000599b74 in __pthread_cond_timedwait(...)
[[  SGX-LKL ]]     #16: 7f603d94ed8f in <unknown>(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 36: tid=54 (7f6000b0c3b0) [java]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000d6471 in futex_wait_queue_me(...)
[[  SGX-LKL ]]     #9: 7f60000d7e19 in futex_wait(...)
[[  SGX-LKL ]]     #10: 7f60000d918b in do_futex(...)
[[  SGX-LKL ]]     #11: 7f60000d9955 in sys_futex(...)
[[  SGX-LKL ]]     #12: 7f600008b86f in lkl_syscall(...)
[[  SGX-LKL ]]     #13: 7f6000597af9 in __filter_syscall6(...)
[[  SGX-LKL ]]     #14: 7f6000597c0f in __timedwait_cp(...)
[[  SGX-LKL ]]     #15: 7f6000599b74 in __pthread_cond_timedwait(...)
[[  SGX-LKL ]]     #16: 7f6000599f16 in pthread_cond_wait(...)
[[  SGX-LKL ]]     #17: 7f603d94eba1 in <unknown>(...)
[[  SGX-LKL ]] -------------------------------------------------------------
[[  SGX-LKL ]] 37: tid=55 (7f6000b0c600) [java]
[[  SGX-LKL ]]     #0: 7f60005b30e1 in __do_futex_sleep(...)
[[  SGX-LKL ]]     #1: 7f60005b3171 in futex_wait(...)
[[  SGX-LKL ]]     #2: 7f60005b32f1 in enclave_futex_timedwait(...)
[[  SGX-LKL ]]     #3: 7f60005b333b in enclave_futex_wait(...)
[[  SGX-LKL ]]     #4: 7f60005d5fd1 in sem_down(...)
[[  SGX-LKL ]]     #5: 7f600008aa93 in __switch_to(...)
[[  SGX-LKL ]]     #6: 7f6000414cfd in __schedule(...)
[[  SGX-LKL ]]     #7: 7f6000414f22 in schedule(...)
[[  SGX-LKL ]]     #8: 7f60000d6471 in futex_wait_queue_me(...)
[[  SGX-LKL ]]     #9: 7f60000d7e19 in futex_wait(...)
[[  SGX-LKL ]]     #10: 7f60000d918b in do_futex(...)
[[  SGX-LKL ]]     #11: 7f60000d9955 in sys_futex(...)
[[  SGX-LKL ]]     #12: 7f600008b86f in lkl_syscall(...)
[[  SGX-LKL ]]     #13: 7f6000597af9 in __filter_syscall6(...)
[[  SGX-LKL ]]     #14: 7f6000597c0f in __timedwait_cp(...)
[[  SGX-LKL ]]     #15: 7f6000599b74 in __pthread_cond_timedwait(...)
[[  SGX-LKL ]]     #16: 7f603d94ed8f in <unknown>(...)
[[  SGX-LKL ]] =============================================================
2020-08-11T11:53:20.000000Z [(H)ERROR] tid(0x7f7f85357700) | :OE_ENCLAVE_ABORTING [/home/sean/code/lsds/sgx-lkl-2020-08-11/openenclave/host/calls.c:oe_call_enclave_function_by_table_id:91]
[   SGX-LKL  ] init (0: 19 exit=0) [   SGX-LKL  ] FAIL: sgxlkl_ethread_init() failed (ethread_id=0 result=19 (OE_ENCLAVE_ABORTING))
Makefile:26: recipe for target 'run-hw' failed
make: *** [run-hw] Error 1
```

**Comment by SeanTAllen** [comment]
@prp @davidchisnall given current priorities, I think this should be reclassified as p2. Agreed? If yes, I'll do as needs triage and put through as proposed p2.

**Comment by prp** [comment]
I think that this issue is caused by https://github.com/lsds/sgx-lkl/issues/680.
